tests/e2e-leg-6/verify-server-logrotate/expected-output/logrotateTimerServiceOut.txt

```diff
@@ -0,0 +1,63 @@
+ clear_data_collector 
+----------------------
+ CLEAR
+(1 row)
+
+        hurry_service         
+------------------------------
+ Service hurried and finished
+(1 row)
+
+     node_name     | success | max_size  | max_age |                               log_file                                | need_rotation | message 
+-------------------+---------+-----------+---------+-----------------------------------------------------------------------+---------------+---------
+ v_vertdb_node0001 | t       | 104857600 | 7       | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | f             | 
+ v_vertdb_node0001 | t       | 104857600 | 7       | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f             | 
+(2 rows)
+
+ALTER DATABASE
+        hurry_service         
+------------------------------
+ Service hurried and finished
+(1 row)
+
+     node_name     | success | max_size  | max_age |                               log_file                                | need_rotation 
+-------------------+---------+-----------+---------+-----------------------------------------------------------------------+---------------
+ v_vertdb_node0001 | t       | 104857600 | 7       | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | f
+ v_vertdb_node0001 | t       | 104857600 | 7       | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f
+ v_vertdb_node0001 | t       |      1024 | 7       | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | t
+ v_vertdb_node0001 | t       |      1024 | 7       | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f
+(4 rows)
+
+ALTER DATABASE
+        hurry_service         
+------------------------------
+ Service hurried and finished
+(1 row)
+
+     node_name     | success | max_size  | max_age  |                               log_file                                | need_rotation 
+-------------------+---------+-----------+----------+-----------------------------------------------------------------------+---------------
+ v_vertdb_node0001 | t       | 104857600 | 7        | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | f
+ v_vertdb_node0001 | t       | 104857600 | 7        | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f
+ v_vertdb_node0001 | t       |      1024 | 7        | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | t
+ v_vertdb_node0001 | t       |      1024 | 7        | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f
+ v_vertdb_node0001 | t       |      1024 | 00:00:01 | /data/vertdb/v_vertdb_node0001_catalog/vertica.log                    | t
+ v_vertdb_node0001 | t       |      1024 | 00:00:01 | /data/vertdb/v_vertdb_node0001_catalog/UDxLogs/UDxFencedProcesses.log | f
```

Comment:
Why does max_age has a different format in the first 4 rows?

Reply:
because we set LogRotateMaxAge to 1sec to rotate log

Reply:
Then why is it 7 in the first 4 rows?

Reply:
because we use default config param first, default Age is 7

Reply:
Link Ref: https://git.verticacorp.com/projects/VER/repos/server/pull-requests/19596/diff#vertica/SQLTest/expected/logrotate_timer_service_test.out